src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -60,6 +60,9 @@
   private boolean m_needToRecordBattleStatistics = true;
   private boolean m_needToCheckDefendingPlanesCanLand = true;
   private boolean m_needToCleanup = true;
+  private boolean needToFindRocketTargets = true;
+  private boolean needToFireRockets = true;
```

Comment:
IDK. In this case it looks like there will be less overall complexity and you would be able to write test code  more easily. The determining factor I think lies in the complexity of the test code. If the variables here make the code easier, then they belong here. Otherwise, if it is hard to test these updates, then moving them would likely be a way to have an easier time writing test code.